sdk/servicebus/azure-servicebus/azure/servicebus/_common/constants.py

```diff
@@ -114,18 +114,21 @@
 TRANSFER_DEAD_LETTER_QUEUE_SUFFIX = '/$Transfer' + DEAD_LETTER_QUEUE_SUFFIX
 
 
-class ReceiveMode(Enum):
-    PeekLock = constants.ReceiverSettleMode.PeekLock
-    ReceiveAndDelete = constants.ReceiverSettleMode.ReceiveAndDelete
+class ServiceBusReceiveMode(Enum):
+    PeekLock = "peeklock"
```

Comment:
Does case matter here -- "PeekLock" vs "peeklock" or this is intentionally to be of lower case.

Reply:
Intentionally lower case.  See the discussion with the architects and the followup I pinged to the messaging archs. last night regarding -mgmt consistency.

Reply:
Closing on this btw, the -mgmt enums are _magical_ and support case-insensitivity on the attribute side.  (Thanks laurent/anna/johan for the confirmation)

https://github.com/Azure/autorest.python/blob/49afdc2f15db14c48c2b7b4a9b37f5c4c7175f73/test/azure/Expected/AcceptanceTests/StorageManagementClient/storage/models/_storage_management_client_enums.py#L12-L35